script.js

```diff
@@ -1,25 +1,199 @@
-function main() {
-  const menu = document.querySelector('#menu-container');
-  const closeButton = document.querySelector('.close-btn');
-  const hamburger = document.querySelector('#humburger-btn');
-  const body = document.querySelector('body');
-  const anchorTags = document.querySelectorAll('.mobile-menu-ul li');
-
-  function openMenuIcon() {
-    menu.classList.add('visible');
-    body.classList.add('overflow-hidden');
-  }
+const menu = document.querySelector('#menu-container');
+const closeButton = document.querySelector('.close-btn');
+const hamburger = document.querySelector('#hamburger-btn');
+const body = document.querySelector('body');
+const anchorTags = document.querySelectorAll('.mobile-menu-ul li');
+
+function openMenuIcon() {
+  menu.classList.add('visible');
+  body.classList.add('overflow-hidden');
+}
+
+function closeMenuIcon() {
+  menu.classList.remove('visible');
+  body.classList.remove('overflow-hidden');
+}
+
+hamburger.addEventListener('click', openMenuIcon);
+closeButton.addEventListener('click', closeMenuIcon);
+anchorTags.forEach((link) => {
+  link.addEventListener('click', closeMenuIcon);
+});
+
+// Menu bar scroll control
 
-  function closeMenuIcon() {
-    menu.classList.remove('visible');
-    body.classList.remove('overflow-hidden');
+// const header = document.getElementById("header");
+// const sticky = header.offsetTop;
+
+// function myFunction() {
+//   if (window.pageYOffset > sticky) {
+//     header.classList.add("sticky");
+//   } else {
+//     header.classList.remove("sticky");
+//   }
+// }
+
+// window.onscroll = function () {
+//   myFunction();
+// };
+
+// Project Objects
```

Comment:
- [ ] Great job so far. I will suggest you remove unused codes from your codebase. They make your codebase to be untidy. We can always use git history to track old codes.

Reply:
Thanks a lot I will do.